app/domain/authentication/usecase/create_token.go

```diff
@@ -12,13 +12,22 @@ func (a Authentication) CreateToken(login authentication.LoginInputs) (string, e
 		return "", err
 	}
 
+	a.logger.Info().
+		Str("CPF terminated in", login.CPF[len(login.CPF)-3:]).
+		Msg("creating token for CPF.")
 	token, err := entities.NewCreateToken(login.Account.Name, login.Account.AccountID)
 	if err != nil {
+		a.logger.Error().Err(err).
+			Str("CPF terminated in", login.CPF[len(login.CPF)-3:]).
+			Msg("error occurred when was trying to create token for CPF.")
 		return "", err
 	}
 
 	err = a.authenticationRepository.CreateToken(token)
 	if err != nil {
+		a.logger.Error().Err(err).
+			Str("CPF terminated in", login.CPF[len(login.CPF)-3:]).
```

Comment:
when putting context info try to stand with snake case, like "CPF terminated in" would be better as "cpf" only. And receiving a string like "********745"